packages/libs/eda/src/lib/core/components/computations/plugins/correlationAssayAssay.tsx

```diff
@@ -182,6 +183,81 @@ export function CorrelationAssayAssayConfiguration(
     });
   }, []);
 
+  // Content for the expandable help section
+  const helpContent = (
+    <div className={cx('-HelpInfoContainer')}>
+      <H6>What is correlation?</H6>
+      <p>
+        The correlation between two variables (taxa, genes, sample metadata,
+        etc.) describes the degree to which their presence in samples
+        co-fluctuate. For example, the Age and Shoe Size of children are
+        correlated since as a child ages, their feet grow.
+      </p>
+      <p>Here we look for correlation between:</p>
+      <ol>
+        <li>Abundance of taxa at a given taxonomic level</li>
+        <li>Abundance of functional data (e.g. pathways, genes)</li>
+      </ol>
+      <br></br>
+      <H6>Inputs:</H6>
+      <p>
+        <ul>
+          <li>
+            <strong>Taxonomic Level.</strong> The taxonomic abundance data to
+            used in the calcualtion
+          </li>
+          <li>
+            <strong>Functional Data.</strong> The pathway, metabolic, or gene
+            data to be correlatd against the taxonomic abundance data.
+          </li>
+          <li>
+            <strong>Method.</strong> The type of correlation to compute. The
+            Pearson method looks for a linear trend in the data, while the
+            Spearman method looks for a monotonic relationship. For Spearman and
+            Pearson correlation, we use the rcorr function from the Hmisc
+            package.
+          </li>
+          <li>
+            <strong>Prevalence Prefilter.</strong> Remove variables that do not
+            have a set percentage of non-zero abundance across samples. Removing
```

Comment:
Is the input a percentage or decimal?

Reply:
It's a percentage now. It gets transformed to a proportion before sending to the backend.